tests/test_apps/test_pictures/test_fetch_pictures.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from server.apps.identity.container import container
+from server.apps.pictures.logic.usecases.pictures_fetch import PicturesFetch
+
+
+def assert_correct_pictures(response, limit) -> None:
+    """Assert pictures."""
+    fields = [
+        'id', 'url',
+    ]
+    assert len(response) == limit
+    for picture in response:
+        for field in fields:
+            assert getattr(picture, field, None)
+
+
+@pytest.mark.timeout(2)
+@pytest.mark.django_db()
+def test_fetch_pictures(
+    user,
+) -> None:
+    """Test fetch pictures work with HTTP."""
+    limit = 5
```

Comment:
nit: из книги «[Чистый код](https://libcat.ru/knigi/kompyutery-i-internet/programmirovanie/396005-14-robert-martin-chistyj-kod-sozdanie-analiz-i-refaktoring.html#text)», глава 2, пункт «Имена должны передавать намерения программиста»: **лучше когда названия переменных, в которых хранятся измеряемые данные, содержат единицу измерения**
```suggestion
    limit_in_sec = 5
```

Reply:
Еще раз спасибо, тоже нравится.